README.md

```diff
@@ -8,6 +8,9 @@ Both ontologies are stored as .obo and .owl files, both of which can be read by
 
 You can browse the terms here: http://www.ebi.ac.uk/ols/ontologies/rxno and here: http://www.ebi.ac.uk/ols/ontologies/mop
 
+## Instructions for contribution
+Follow the instructions in the documentation folder (https://github.com/rsc-ontologies/rxno/tree/master/docs) and use the flowcharts provided there to add new reactions.
```

Comment:
can be made more clean with 
`[documentation folder](docs)` which when Merged with link to docs dir

[documentation folder](docs/)

Reply:
thanks for the hint, yes that is so indeed better